pkgs/applications/misc/archivebox/default.nix

```
@@ -22,6 +22,9 @@ let
       sha256 = "sha256-Ms55Lum2oMu+w0ASPiKayfdl3/jCpK6SR6FLK6OjZac=";
     };
   });
+  django-extensions' = django-extensions.overrideAttrs (oldAttrs: rec {
+    propagatedBuildInputs = [];
+  });
```

```suggestion
  django-extensions' = django-extensions.override { django = django_3'; };
```
otherwise we might clear other dependencies